fit-vids.js

```diff
@@ -1,27 +1,36 @@
-class FitVids extends HTMLElement {
-  connectedCallback() {
-    const videoSources = ['iframe[src*="youtube"]', 'iframe[src*="vimeo"]'];
-    this.style.display = "block";
+export class FitVids extends HTMLElement {
+  constructor() {
+    super();
+    this.attachShadow({ mode: "open" });
+    this.shadowRoot.innerHTML = `
+      <style>
+        :host {
+          display: block;
```

Comment:
Would this be a good use case for `display: contents` as you're not really sizing the `<fit-vids>` element, so much as the `iframe` inside of it?

Reply:
The `block` is there to make it not inline by default but also to remove the little bit of whitespace line-height from inline elements. `display: contents` could work here, but I don't quite trust it [from an a11y perspective yet](https://ericwbailey.website/published/display-contents-considered-harmful/)... but it's probably right for this situation. Will keep an eye on it or PR something to play with after this gets merged.